README.md

```diff
@@ -123,10 +73,10 @@ void redisClusterFree(redisClusterContext *cc);
 
 ### Cluster connecting
 
-The function `redisClusterContextInit` is used to create a so-called `redisClusterContext`. 
-The function `redisClusterSetOptionAddNodes` is used to add the redis cluster address. 
-The function `redisClusterConnect2` is used to connect to the redis cluser. 
-The context is where Hiredis-vip Cluster holds state for connections. The `redisClusterContext`
+The function `redisClusterContextInit` is used to create a so-called `redisClusterContext`.
+The function `redisClusterSetOptionAddNodes` is used to add the redis cluster address.
+The function `redisClusterConnect2` is used to connect to the redis cluser.
+The context is where the state for connections are hold. The `redisClusterContext`
```

Comment:
"are hold" => "is held" or "is kept". (This refers to *the state* which is in singular.)

Reply:
done